docker/openSUSE_Leap.dockerfile

```diff
@@ -0,0 +1,58 @@
+# WARNING! This is not in any way production ready. It is just for testing!
+FROM registry.opensuse.org/opensuse/leap:15.2
+
+# ENV Variables we are using.
+ENV container docker
+ENV DISTRO SUSE
+
+# Update Leap to most current packages
+RUN ["zypper", "-n", "update"]
+
+# Install runtime dependencies for Cobbler
+RUN ["zypper", "-n", "in", "python3", "python3-devel", "python3-pip", "python3-setuptools", "python3-wheel", \
+    "python3-distro", "python3-future", "python3-coverage", "apache2", "apache2-devel", "acl", \
```

Comment:
in all other distros you remove python3-future and here it is still being installed?

Reply:
Done